dm/tests/openapi/client/openapi_task_check

```diff
@@ -413,6 +413,26 @@ def get_task_status_success(task_name, total):
     print("get_task_status_success resp=", data)
     assert data["total"] == int(total)
 
+def get_task_io_counter_success(task_name, total):
+    url = API_ENDPOINT + "/" + task_name + "/status"
+    resp = requests.get(url=url)
+    data = resp.json()
+    assert resp.status_code == 200
+    print("get_task_io_counter_success resp=", data)
+    assert data["total"] == int(total)
+
+    for item in data.get("data", []):
+        unit = item.get("unit")
+        sync_status = item.get("sync_status", {})
+        # Only validate I/O statistics if the unit is "Sync"
+        if unit == "Sync":
+            dump_io_total_bytes = sync_status.get("dump_io_total_bytes")
+            io_total_bytes = sync_status.get("io_total_bytes")
+            assert dump_io_total_bytes is not None and io_total_bytes is not None
+            assert dump_io_total_bytes > 0 and io_total_bytes > 0
```

Comment:
check > 0 is not enough, we need to make sure it's right, cloud needs it to get a billing

Reply:
any suggestions to validate the exactly TCP IO usage with this two fields🤔?

Reply:
you can run a test locally, see and check the value meets our expectations. if the number varies depends on environment, a range check wound be fine

Reply:
I find it depends on sync progress, sometime it have a little bit gap. So verify in a rangehttps://github.com/pingcap/tiflow/pull/11742/files#diff-e9ab88b6967b80315b97f736b2bb8f4fe37e2b35b98c5f27cf667c011209c9ffR435-R437